### DISCUSSION TITLE
What is the default master password?

### DISCUSSION THREAD

**Discussion by Curry0925** [kingToolbox/WindTerm#861] [Q&A]
安装以后，连接窗口会话时间过长提示锁屏密码，这个密码是有默认的吗，我在安装过程中没有这个设置

**Comment by kingToolbox** [comment]
The default master password is an empty string, which means you can just press `Enter`. For more information, you can refer to [about protection](https://kingtoolbox.github.io/tags/protection/).

If you want to turn it off then please set `Menubar - Session - Preferences - Settings - Security - Lock screen timeout` to `0 minute`.
